profiler/lib.profiler.common/nbproject/org-netbeans-lib-profiler-common.sig

```diff
@@ -384,7 +384,6 @@ fld public final static java.lang.String PLATFORM_MAC_OS
 fld public final static java.lang.String PLATFORM_SOLARIS_AMD64_OS
 fld public final static java.lang.String PLATFORM_SOLARIS_INTEL_OS
 fld public final static java.lang.String PLATFORM_SOLARIS_SPARC64_OS
-fld public final static java.lang.String PLATFORM_SOLARIS_SPARC_OS
```

Comment:
One last request: Could we rework this, so that compatibility is retained? This should be binary compatible, as "final static"s are resolved at compile time and not runtime, but it is an source incompatible change. Is there harm in retaining the string?